com.transmissionbt.Transmission.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id": "com.transmissionbt.Transmission",
+    "branch": "master",
```

Comment:
You should set it to `stable` or remove the line altogether as it's assumed by buildbot anyway.